### DISCUSSION TITLE
ESP32 und langsames WebUI

### DISCUSSION THREAD

**Discussion by Gnarzkopp** [jomjol/AI-on-the-edge-device#2578] [Ideas]
Hallo,
begeistert von dem projekt habe ich mir gleich einen ESP32 CAM bestellt ( https://amzn.eu/d/1P7eIWX ). Musste dann aber leider feststellen, dass das Teil nicht funktioniert. Das Webinterface wurde gar nicht bis sehr langsam geladen, die Einrichtung konnte nicht abgeschlossen werden.
Diverse Hinweise gaben ein Problem mit der ESP32 Rev.3 aus, welche laut meiner Suche behoben sein sollten. Also den ESP32 getauscht  und nochmal getauscht und fast frustriert aufgegeben.
Dann habe ich aber den ESP32 direkt an ein Netzteil angeschlossen (ohne den USB to Serial Adapter und siehe da, es funktioniert ohne Probleme.

LÖSUNG: zum flashen funktioniert der Adapter, für den laufeneden Betrieb, muss aber die Stromversorgung direkt an den ESP32.
Keine Ahnung wie es sich mit den Adaptern, die man mit Kabel verbindet verhält. Das Problem trat bei mir mit den im Link https://amzn.eu/d/1P7eIWX gezeigten Adaptern auf.

**Comment by caco3** [comment]
Schön, dass es auch bei dir nun klappt.

Dass es einen Unterschied macht, ob via USB oder direkt eingespiesen wird, konnte ich bis jetzt noch nie feststellen. Ev. war deine USB_Speisung ungenügend?

Mit dem v3 hatten andere auch schon Probleme. Einige sagten auch, dass es abhängig vom Wifi-Kanal ist (Siehe https://github.com/jomjol/AI-on-the-edge-device-docs/issues/11).

**Comment by Zenyte-zz** [comment]
Genau das selbe Phänomen hatte ich auch, erst als ich das usb 2 serial abgezogen und die Stromversorgung direkt auf den esp32 gelötet habe funktioniert es 1A.
